metrics/encoder.go

```diff
@@ -0,0 +1,67 @@
+package metrics
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Encoder is the signature for encoders
+type Encoder func(name string, op Op, value interface{}, tags Tags, rate float64) (string, error)
+
+// StdoutEncoder is a simple encoder to be used to write to stdout.
+func StdoutEncoder(name string, op Op, value interface{}, tags Tags, rate float64) (string, error) {
+	return fmt.Sprintf("METRIC: %s | %d | %v | %v | %f\n", name, op, value, tags, rate), nil
+}
+
+// StatsDEncoder implements statsd protocol
+func StatsDEncoder(name string, op Op, value interface{}, tags Tags, rate float64) (string, error) {
+	ct := make([]string, len(tags))
+	for k, t := range tags {
+		ct[k] = fmt.Sprintf("%v:%v", t.Key, t.Value)
+	}
+	st := strings.Join(ct, ",")
+
+	switch op {
+	case OpCounterAdd:
+		return fmt.Sprintf("%s:%v|c|@%.4f|#%s\n", name, value, rate, st), nil
+	case OpGaugeUpdate:
+		return fmt.Sprintf("%s:%v|g|@%.4f|#%s\n", name, value, rate, st), nil
+	case OpHistogramUpdate:
+		return fmt.Sprintf("%s:%v|h|@%.4f|#%s\n", name, value, rate, st), nil
+	case OpEventSend:
```

Comment:
I've created a new issue https://github.com/socialpoint-labs/bsk/issues/22 since I think is out of the scope of this PR which is just about importing from the private bsk to this one.